app/views/quby/v1/single_page/_panel.html.haml

```diff
@@ -0,0 +1,64 @@
+/ Begin _panel.html.haml
```

Comment:
Ja, het verschil is minimaal, maar hij wordt gebruikt. Om duplicatie te voorkomen ga ik kijken of ik het kan overbruggen met een regeltje javascript om de hidden class van alle panels af te halen in de single page mode.